test/known_problems/should_fail/binary_comprehension.erl

```diff
@@ -0,0 +1,10 @@
+-module(binary_comprehension).
+-compile([debug_info]).
+-export([
+         bitstring_fun/0
+        ]).
+
+-spec bitstring_fun() -> <<_:7, _:_*3>> | string().
+bitstring_fun() ->
+    << <<1:N>> || N <- lists:seq(3, 12)>>.
+
```

Comment:
"fun" means lambda expression in Erlang, so that part isn't very good in the function name.